--- origsrc/skype4pidgin-1.2.1/skypeweb/CMakeLists.txt	2016-08-08 02:32:12.000000000 -0500
+++ src/skype4pidgin-1.2.1/skypeweb/CMakeLists.txt	2016-09-19 12:13:13.237342100 -0500
@@ -37,7 +39,7 @@ set(SRC_LIST
     libskypeweb.c 
     )
 
-add_library(${PROJECT_NAME} SHARED ${SRC_LIST})
+add_library(${PROJECT_NAME} MODULE ${SRC_LIST})
 
 target_link_libraries(${PROJECT_NAME}
     ${PURPLE_LIBRARIES}
